clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -285,24 +310,24 @@ private EhcacheEntityResponse invokeLifeCycleOperation(ClientDescriptor clientDe
       case DESTROY_SERVER_STORE:
         destroyServerStore(clientDescriptor, (DestroyServerStore) message);
         break;
+      case CREATE_CLUSTERED_MAP:
+        createConcurrentMap(clientDescriptor, (CreateClusteredMap) message);
```

Comment:
I understand creation of map is happening in single round trip to server.  Only once the create map is called per map.The default serializer uses only one map. Right @albinsuresh ?